src/components/DataDecorator/Decorator/Decorator.stories.js

```diff
@@ -27,18 +27,38 @@ const storyProps = () => ({
     min: scoreThresholds[0],
     range: true,
   }),
-  href: text('Link (`href`)', href),
-  onClick: action('onClick'),
-  inert: boolean('Non-interactive (`inert`)', false),
   active: boolean('Active (`active`)', false),
   noIcon: boolean('No icon (`noIcon`)', false),
 });
 
 storiesOf(components('Decorator'), module)
-  .add('Default', () => <Decorator {...storyProps()} />)
+  .add('Default', () => (
+    <>
+      <p className="bx--type-body-long-01">This Decorator is inert.</p>
+      <p>
+        <Decorator {...storyProps()} />
+      </p>
+    </>
+  ))
   .add('Inline', () => (
     <p className="bx--type-body-long-01">
       This is an inline decorator <Decorator {...storyProps()} inline /> that
       appears alongside some text.
     </p>
+  ))
+  .add('with href', () => (
+    <>
+      <p className="bx--type-body-long-01">This Decorator is a link.</p>
+      <p>
+        <Decorator {...storyProps()} href={text('Link (`href`)', href)} />
+      </p>
+    </>
+  ))
+  .add('with onClick', () => (
+    <>
+      <p className="bx--type-body-long-01">This Decorator is a button.</p>
+      <p>
+        <Decorator {...storyProps()} onClick={action('onClick')} />
+      </p>
+    </>
```

Comment:
Because the visual distinctions here can sometimes be unclear unless you inspect or hover over a given element, I think it is best to say which variation is being displayed.